viewer/viewer-awt/src/main/java/org/icepdf/ri/common/utility/search/SearchFilterButtonWrapper.java

```diff
@@ -0,0 +1,226 @@
+package org.icepdf.ri.common.utility.search;
+
+import org.icepdf.ri.common.DropDownButton;
+import org.icepdf.ri.common.PersistentJCheckBoxMenuItem;
+import org.icepdf.ri.common.SwingViewBuilder;
+import org.icepdf.ri.common.views.Controller;
+import org.icepdf.ri.images.Images;
+import org.icepdf.ri.util.PropertiesManager;
+
+import javax.swing.*;
+import java.util.ResourceBundle;
+import java.util.prefs.Preferences;
+
+import static org.icepdf.ri.util.PropertiesManager.*;
+
+public class SearchFilterButtonWrapper {
```

Comment:
Suggest just calling this SearchFilterButton and have it extend DropDownButton.  Extending DropDownButton will get rid of the the wrapper.getButton() calls in the SearchToolBar and SearchPanel.